tests/queries/0_stateless/02481_async_insert_dedup.python

```diff
@@ -81,7 +81,7 @@ EventDate DateTime,
 KeyID     UInt32
 ) Engine = ReplicatedMergeTree('/clickhouse/tables/{shard}/{database}/t_async_insert_dedup', '{replica}')
 PARTITION BY toYYYYMM(EventDate)
-ORDER BY (KeyID, EventDate)
+ORDER BY (KeyID, EventDate) SETTINGS use_async_block_ids_cache = 1
```

Comment:
Should not we have tests for async with and without this feature?

Reply:
Yeah we should. actually we should also check we could filter multiple block ids at one time (I can observe it happens in the log). Any suggestions?

Reply:
I didn't read this test thoroughly, but we can just duplicate it with different settings (or have duplicate tables within the same test).

Reply:
I have an idea: record `async inserts retry times` in *ProfileEvents* . We can assume the retry times will be less when enable async cache.

Reply:
But I don't know whether we can collect retry times for one specific query. I'm not sure if this idea can work or not